components/list/test/list.test.js

```diff
@@ -567,7 +578,7 @@ describe('d2l-list-item', () => {
 			const el = await fixture(html`
 				<d2l-list-item action-href="javascript:void(0)" label="item">
 					<div>Item 1</div>
-					<div slot="secondary"><d2l-button>Button</d2l-button></div>
+					<div><d2l-button>Button</d2l-button></div>
```

Comment:
I noticed that tests weren't failing as expected when the flag was off so I did some tweaking to the fixtures.